freqtrade/freqtradebot.py

```diff
@@ -579,7 +580,8 @@ def execute_entry(self, pair: str, stake_amount: float, price: Optional[float] =
             exchange=self.exchange.id,
             open_order_id=order_id,
             strategy=self.strategy.get_strategy_name(),
-            buy_tag=buy_tag,
+            # TODO-lev: compatibility layer for buy_tag (!)
```

Comment:
What does this mean?

Reply:
what it says? that we'll need a compatibility layer for the buy-tag field (not necessarily here, but probably in persistence).
Strategies use this field as part of callbacks to determine the "matching" sell strategy - simply renaming the field to `entry_tag` (or whatever else) will break people's strategies - so it'll need a compatibility layer.